docs/CONTRIBUTING.md

```diff
@@ -81,13 +81,11 @@ We encourage everyone to participate in the review process to get familiar with
 - Is the code easy to understand and to maintain?
 - Is there sufficient detail to inform any changes to documentation?
 
-A maintainer does a thorough code review and when satisfied with the code, accepts and may label the pull request as `needs testing` to indicate the need for a manual [black-box testing](https://en.wikipedia.org/wiki/Black-box_testing) pass. A pull request may have `needs testing` removed and added again several times until the behavior is thoroughly verified. Once the behavior has been verified, the pull request is labelled as `behavior verified`.
+A maintainer does a thorough code review and when satisfied with the code, accepts and merges. If it's felt that the PR needs a manual [black-box testing](https://en.wikipedia.org/wiki/Black-box_testing) pass, the `needs testing` label should be added after merging.
```

Comment:
> If it's felt that the PR needs a manual [black-box testing](https://en.wikipedia.org/wiki/Black-box_testing) pass, the `needs testing` label should be added after merging

In our case it's rather rare to have a pr that does not need testing I would say over 90% of them are verified manually.
The sentence above indicates it's the other way around. Maybe something like:
> Typically, manual testing is a requirement and then the `needs testing` label should be added after merging. Only in rare cases do we merge pull requests that are deemed safe enough to skip additional testing.